presto-docs/src/main/sphinx/connector/hive.rst

```diff
@@ -876,4 +897,18 @@ Hive Connector Limitations
 
 * :doc:`/sql/delete` is only supported if the ``WHERE`` clause matches entire partitions.
 * :doc:`/sql/alter-schema` usage fails, since the Hive metastore does not support renaming schemas.
+* Writing to and creation of transactional tables is not supported.
+* Reading ORC ACID tables created with Hive Streaming ingest is not supported.
+
+Hive 3 Related Limitations
+^^^^^^^^^^^^^^^^^^^^^^^^^^
 
+* For security reasons, ``sys`` system catalog is not accessible in Presto.
+* Hive's ``timestamp with local zone`` data type is not supported in Presto.
+  It is possible to read from a table having a column of this type, but the column
+  itself will not be accessible. Writing to such a table is not supported.
+* Due to Hive issues `HIVE-21002 <https://issues.apache.org/jira/browse/HIVE-21002>`_,
+  `HIVE-22167 <https://issues.apache.org/jira/browse/HIVE-22167>`_, Presto does
+  not correctly read ``timestamp`` values from Parquet, RCFile with binary serde and
```

Comment:
binary serde ? Do you mean https://cwiki.apache.org/confluence/display/Hive/SerDe ?

Reply:
`LazyBinarySerDe`. I think it's clear as is

Reply:
ok